src/www/ui/page/UploadVcsPage.php

```diff
@@ -161,20 +161,8 @@ protected function handleUpload(Request $request)
       return array(false, _($ErrorMsg), $description);
     }
 
-    AgentCheckBoxDo($jobpk, $uploadId);
-
-    $msg = "";
-    /** check if the scheudler is running */
-    $status = GetRunnableJobList();
-    if (empty($status)) {
-      $msg .= _("Is the scheduler running? ");
-    }
-    $Url = Traceback_uri() . "?mod=showjobs&upload=$uploadId";
-    $text = _("The upload");
-    $text1 = _("has been queued. It is");
-    $msg .= "$text $Name $text1 ";
-    $keep =  "<a href='$Url'>upload #" . $uploadId . "</a>.\n";
-    return array(true, $msg.$keep, $description, $uploadId);
+    $message = $this->postUploadAddJobs($request, $Name, $uploadId);
```

Comment:
Please add `$jobpk` as the 4th parameter to prevent additional job being created.

Reply:
Thanks @GMishx, Actually this solved the issue of scheduling agents with different name(once as `wget` and other with the `file name`). This issue was due to duplicate job created in `postUploadAddJobs` function as '$jobpk' wasn't passed.